src/main/xerus/monstercat/tabs/TabPlaylist.kt

```diff
@@ -0,0 +1,103 @@
+package xerus.monstercat.tabs
+
+import javafx.beans.Observable
+import javafx.beans.value.ObservableValue
+import javafx.scene.control.*
+import javafx.scene.input.KeyCode
+import javafx.scene.input.MouseButton
+import javafx.util.Callback
+import xerus.ktutil.javafx.MenuItem
+import xerus.ktutil.javafx.fill
+import xerus.ktutil.javafx.properties.ImmutableObservable
+import xerus.ktutil.javafx.properties.listen
+import xerus.monstercat.api.Player
+import xerus.monstercat.api.Playlist
+import xerus.monstercat.api.response.Track
+
+
+class TabPlaylist : VTab() {
+	var table = TableView<Track>().apply {
+		columnResizePolicy = TableView.CONSTRAINED_RESIZE_POLICY
+	}
+	
+	init {
+		table.items = Playlist.tracks
+		
+		table.columns.addAll(TableColumn<Track, String>("Artists").apply {
```

Comment:
I'm pretty sure I have a util function to create these columns with a lot less code.

Reply:
Well it's actually not that large of a block of code, only 8 lines. Couldn't find that function either

Reply:
Found it ! a3d188163935d19426959611a8fce4191e87e858